src/Entities/Application.php

```diff
@@ -87,7 +90,7 @@ public function getExperience(): string
         return $this->experience;
     }
 
-    public function getDiversitech(): bool
+    public function getDiversitech(): int
```

Comment:
The `bool`s should stay `bool`, revert this change.

Reply:
Done